playground/src/index.css

```diff
@@ -3,3 +3,34 @@ html, body {
   height: 100%;
   font-family: Tahoma, sans-serif;
 }
```

Comment:
why is this raw CSS not a module.sass?

Reply:
This project doesn't have a module.css processor yet. We can add that in a separate change, though.

---
In reply to: [224212334](https://github.com/Microsoft/tsdoc/pull/100#discussion_r224212334) [](ancestors = 224212334)